powerauth-test-server/src/main/resources/application.properties

```diff
@@ -15,6 +15,16 @@ spring.datasource.username=sa
 spring.datasource.password=
 spring.datasource.driver-class-name=org.h2.Driver
 
+# PostgreSQL configuration
+#spring.datasource.url=jdbc:postgresql://localhost:5432/powerauth
+#spring.datasource.username=powerauth
+#spring.datasource.password=
+#spring.datasource.driver-class-name=org.postgresql.Driver
+#spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
We should probably use `PostgreSQL95Dialect` in the example, `PostgreSQLDialect` is deprecated.